recipes/cf-nvidia-tools/recipe.yaml

```diff
@@ -0,0 +1,46 @@
+context:
+  name: cf-nvidia-tools
+  version: "1.0.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - path: bin
+    target_directory: bin
+
+build:
+  skip:
+    - not linux
+  noarch: generic
+  script:
+    - ls -lah
+    - cp -r bin $PREFIX/bin
+  number: 0
+
+requirements:
+  run:
+    - __unix
+    - patchelf >=0.12
+
+tests:
+  - script:
```

Comment:
Would it make sense to have a `shellcheck` step here?

Reply:
Never heard of shellcheck before now. I'll add it.